Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -1,12 +1,14 @@
 // Copyright 2008 Dolphin Emulator Project
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-License-Identifier: GPL-2.0f-or-later
```

Comment:
This doesn't seem to be a valid SPDX identifier, so I'm assuming it's a typo?

Reply:
i was converting doubles to floats :/